app/src/services/market_maker.ts

```diff
@@ -66,7 +68,12 @@ class MarketMakerService {
     logger.log(`Add funding to market maker ${amount}`)
 
     try {
-      return this.contract.addFunding(amount, distributionHint)
+      const overrides = {
+        gasLimit: 750000,
```

Comment:
Was this part of a test or do we need to hardcode the gas limit so high?

Reply:
We need the gas limit high, also It throws an error

Reply:
I remove the gas limit, and tested again with Trust Wallet and Pillar Wallet, and is working fine! So I think this was added to solve a problem with gnosis safe